Ollivanders/src/net/pottercraft/ollivanders2/spell/APPARATE.java

```diff
@@ -32,6 +32,11 @@
  */
 public final class APPARATE extends O2Spell
 {
+    /**
+     * The maximum apparate distance, value of 0 or less means no limit
+     */
+    public static int maxApparateDistance = 0;
```

Comment:
Add a static variable on the APPARATE spell for the max distance, defaulted to "off". When a config is unset, it will also have the value "0".